modules/nf-core/pbbam/pbmerge/tests/main.nf.test

```diff
@@ -0,0 +1,64 @@
+nextflow_process {
+
+    name "Test Process PBBAM_PBMERGE"
+    script "../main.nf"
+    config "./nextflow.config"
+    process "PBBAM_PBMERGE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pbbam"
+    tag "pbbam/pbmerge"
+
+    test("PacBio pbbam - Merging Bam test") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    [
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/pacbio/bam/alz.ccs.fl.NEB_5p--NEB_Clontech_3p.flnc.clustered.bam'           , checkIfExists: true),
+                        file(params.modules_testdata_base_path + 'genomics/homo_sapiens/pacbio/bam/alz.ccs.fl.NEB_5p--NEB_Clontech_3p.flnc.clustered.singletons.bam', checkIfExists: true)
+                    ]
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.bam).match("bam") },
+                { assert snapshot(process.out.pbi).match("pbi") },
+                { assert path(process.out.versions.get(0)).text.contains("pbbam") }
```

Comment:
```suggestion
                { assert snapshot(process.out.versions).match("version") }
```

Reply:
This might fix the linting. SO change that line first, then delete the snapshot and run the test locally. Test with linting and push afterwards.